app/models/endpoint.rb

```diff
@@ -3,6 +3,7 @@ class Endpoint < ApplicationRecord
 
   default_value_for :verify_ssl, OpenSSL::SSL::VERIFY_PEER
   validates :verify_ssl, :inclusion => {:in => [OpenSSL::SSL::VERIFY_NONE, OpenSSL::SSL::VERIFY_PEER]}
+  validates :port, :numericality => {:only_integer => true, :allow_nil => true, :greater_than => 0}
```

Comment:
Good thing to know I'm at least consistent with myself 😆 

https://github.com/ManageIQ/manageiq/issues/8241#issuecomment-216568970